tasks/queue-granules/tests/index.js

```diff
@@ -830,3 +837,217 @@ test('updatedGranuleBatchCreatedAt updates batch granule object with correct cre
   const actual = updateGranuleBatchCreatedAt(testGranuleBatch, createdAtTestDate);
   t.deepEqual(actual, expected);
 });
+
+test.serial('hitting defaults for branch coverage', async (t) => {
+  const dataType = `data-type-${randomString().slice(0, 6)}`;
+  const version = '6';
+  const collectionConfig = { foo: 'bar' };
+  await t.context.getCollection.withArgs({
+    prefix: t.context.stackName,
+    collectionName: dataType,
+    collectionVersion: version,
+  }).returns(collectionConfig);
+
+  const { event } = t.context;
+
+  event.config.concurrency = 99;
+
+  event.input.granules = undefined;
+
+  await queueGranules(event);
+
+  t.is(pMapSpy.getCalls().length, 1);
+});
+
+test.serial('does not change collection id on granule', async (t) => {
+  const { event } = t.context;
+
+  event.input.granules = [
+    {
+      granuleId: 'granule-1',
+      dataType: 'http_testcollection_test-stackName-DiscoverGranules-1686092642035',
+      version: '001',
+      files: [],
+    },
+    {
+      granuleId: 'granule-2',
+      dataType: 'http_testcollection_test-stackName-DiscoverGranules-1686092642035',
+      version: '001',
+      files: [],
+    },
+    {
+      granuleId: 'granule-3',
+      dataType: 'http_testcollection_test-stackName-DiscoverGranules-1686092642035',
+      version: '001',
+      files: [],
+    },
+  ];
+  const enqueueGranuleIngestMessageMock = sinon.spy((params) => params);
+
+  const testMocks = {
+    updateGranuleMock: sinon.spy(async () => { }),
+    enqueueGranuleIngestMessageMock,
+  };
+
+  await queueGranules(event, testMocks);
+
+  const createdMap = Object.fromEntries(
+    testMocks.updateGranuleMock.getCalls()
+      .map(({ args: [params] }) => [params.granuleId, params.body.createdAt])
+  );
+  t.deepEqual(
+    testMocks.updateGranuleMock.getCalls()
+      .map(({ args: [params] }) => params)
+      .sort(({ granuleId: a }, { granuleId: b }) => a.localeCompare(b)),
```

Comment:
What is the benefit of sorting? 🤔

Reply:
What if we only compare the collectionId ?
Something like:
```
t.deepEqual(
    testMocks.updateGranuleMock.getCalls()
      .map(({ args: [{ collectionId }] }) => collectionId),
    event.input.granules.map(({ granuleId, dataType, version }) => { const collectionId = constructCollectionId(dataType, version); return collectionId })
  );
```

Reply:
Isn't the assertion about all the params more pedantic? I sort because the calls can be reordered by the chunking/grouping and deepEqual cares about the ordering I think.